docs/build-docs.sh

```diff
@@ -8,7 +8,7 @@ set -o nounset
 action=$1
 
 if [ "$action" == "linkcheck" ]; then
-  sphinx-build -WETan -j auto -D language=en -b linkcheck -d docs/build/doctrees docs/source docs/build/linkcheck
```

Comment:
Are there some warnings we should be aware of?

Reply:
Fear not, i'll revert this before I merge (it got committed by mistake!). While I'm working locally it just means I get a list of all the issues like broken anchor links and links to moved pages in one hit.